src/components/data-table-v2/_data-table-v2-core.scss

```diff
@@ -170,3 +170,296 @@
     background-color: rgba($brand-02, 0.1);
   }
 }
+
+@mixin data-table-v2-core--x {
+  //================================================
+  //MAIN CONTAINER
+  //================================================
+  .#{$prefix}--data-table-v2-container {
+    min-width: rem(600px);
```

Comment:
Do we need this on `data-table-v2` versus the container?

Reply:
v9 originally had a min-width in the container, which is why I had that here. I'm not sure if that was for a specific reason.